app/api/sessions.py

```diff
@@ -59,8 +59,9 @@ def before_post(self, args, kwargs, data):
         ):
             raise ForbiddenError({'pointer': ''}, "Sessions are disabled for this Event")
 
+        excluded = ['track']
```

Comment:
But we want it to be required when the organizers have marked it required

Reply:
how to check if any custom form field is required?

Reply:
I have a doubt, when other custom form fields are made required they only have frontend validation or for each required field the validation in server

Reply:
Because for track also, frontend validation is working while it is made required.

Reply:
They have server validation as well

Reply:
how to check that track is required or not on server side?

Reply:
By not passing track in excluded